packages/layout/header.jade

```diff
@@ -1,2 +1,11 @@
-template(name='header')
-  p HEADER
\ No newline at end of file
+template(name="header")
+  header
+    nav.navbar
+      .fluid-container
+        .container
```

Comment:
Removed. The padding seems okay so i probably corrected that issue in EIDR and maybe Rana with custom styles. I'll fix it in the others.